api/src/pcapi/sandboxes/scripts/utils/helpers.py

```diff
@@ -17,19 +20,19 @@ def get_booking_helper(booking):  # type: ignore [no-untyped-def]
     )
 
 
-def get_offer_helper(offer):  # type: ignore [no-untyped-def]
+def get_offer_helper(offer: offers_models.Offer) -> dict:
     return dict(
         as_dict(offer),
         **{
             "venueCity": offer.venue.city,
             "venueName": offer.venue.name,
             "thingName": offer.product.name,
-            "status": offer.status.name,
+            "status": offer.status.name,  # type: ignore [attr-defined]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏on a pas forcément de `offer.status.name` ici ? 
Est ce que un `offer.status.name if offer.status else None` conviendrait ici ?

Reply:
```
    @hybrid_property
    def status(self) -> OfferStatus:
```
ne renvoie pas `None`, c'est un `type: ignore` lié aux `hybrid_property` comme on en a beaucoup (mypy considère `status` comme de type `Callable`)